### PR TITLE
fix flake8 errors

### PR DESCRIPTION
**Pull request by mhrabovcin**

### CODE REVIEW

File: tools/airgap_linter.py [hunk 1]
```diff
@@ -127,10 +127,10 @@ def validate_images(framework_directory):
         for line in lines:
             line = line.strip()
             if "image:" in line:
-                image_matcher = re.compile("image:\s?(.*)$", re.IGNORECASE)
+                image_matcher = re.compile(r"image:\s?(.*)$", re.IGNORECASE)
                 match = image_matcher.match(line)
                 image_path = match.group(1)
-                env_var_matcher = re.compile("[\"]?\{\{[A-Z0-9_]*\}\}[\"]?")
+                env_var_matcher = re.compile(r"[\"]?\{\{[A-Z0-9_]*\}\}[\"]?")
```
Comment (porridge) [inline tools/airgap_linter.py]: I think you changed semantics here.
```
>>> r"[\"]?\{\{[A-Z0-9_]*\}\}[\"]?" == "[\"]?\{\{[A-Z0-9_]*\}\}[\"]?"
False
>>> 
```

File: tools/airgap_linter.py [hunk 2]
```diff
@@ -127,10 +127,10 @@ def validate_images(framework_directory):
         for line in lines:
             line = line.strip()
             if "image:" in line:
-                image_matcher = re.compile("image:\s?(.*)$", re.IGNORECASE)
+                image_matcher = re.compile(r"image:\s?(.*)$", re.IGNORECASE)
                 match = image_matcher.match(line)
                 image_path = match.group(1)
-                env_var_matcher = re.compile("[\"]?\{\{[A-Z0-9_]*\}\}[\"]?")
+                env_var_matcher = re.compile(r'["]?\{\{[A-Z0-9_]*\}\}["]?')
```
Comment (adamtheturtle) [inline tools/airgap_linter.py]: @porridge I hope that your concern here is now addressed.
@mhrabovcin I hope that your linters still run.

Comment (porridge) [reply tools/airgap_linter.py]:
```
>>> "[\"]?\{\{[A-Z0-9_]*\}\}[\"]?" == r'["]?\{\{[A-Z0-9_]*\}\}["]?'
True
>>> 
```
I'm happy!

**Comment by porridge** [comment]
Force-merging as the failures are unrelated flakes.

**Pull request merged**